graylog2-web-interface/src/components/common/Icon.tsx

```diff
@@ -78,7 +79,13 @@ type Props = {
   type?: IconTypes,
   fixedWidth?: boolean,
   inverse?: boolean,
-  style?: React.CSSProperties
+  style?: React.CSSProperties,
+  onClick?: (event: SyntheticEvent) => void,
+  onMouseEnter?: (event: SyntheticEvent) => void,
+  onMouseLeave?: (event: SyntheticEvent) => void,
+  onFocus?: (event: SyntheticEvent) => void,
```

Comment:
These type definitions are correct, but we can define slightly more specific type definitions here:
```
  onClick?: (event: React.MouseEvent<SVGSVGElement>) => void,
  onMouseEnter?: (event: React.MouseEvent<SVGSVGElement>) => void,
  onMouseLeave?: (event: React.MouseEvent<SVGSVGElement>) => void,
  onFocus?: (event: React.FocusEvent<SVGSVGElement>) => void,
```